@here/harp-mapview/lib/text/Placement.ts

```diff
@@ -672,6 +681,23 @@ function placePointLabelAtAnchor(
         // It might be changed if we would like to render text without icon (at border, etc.).
         return persistent ? PlacementResult.Rejected : PlacementResult.Invisible;
     }
+
+    if (measureText) {
+        // Bigger label margin is applied only for the first time it is placed and its visible
+        // with its normal bounds. This margin compensates the changes of label bounds due to
+        // distance scaling and it is designed to cover size changes due to camera panning.
+        // This ways label "reserves" enough space so it may "grow" without affecting other
+        // labels, no collisions occurs and thus limits labels flickering on pan.
```

Comment:
This comments tells basically the same as the comment for the margin percentage constant. So just leave one of the two, and specify that the extra margin is not only to solve flickering on panning with distance scaling, it's also useful for other scenarios where small camera changes would cause collisions between visible labels that are too close.

Reply:
Ok, I have removed that one, but replaced it with some explanation why it is after visibility check. I think it is crucial and may be not so clear to everybody.